skimage/feature/__init__.py

```diff
@@ -28,6 +29,16 @@
                    draw_haar_like_feature)
 
 
+@deprecated(alt_func='skimage.registration.register_translation',
+            removed_version='0.19')
+def register_translation(src_image, target_image, upsample_factor=1,
+                         space="real", return_error=True):
+    from ..registration._phase_cross_correlation import (
+        phase_cross_correlation as func)
```

Comment:
Why as func???

Reply:
Because func is short, func is universal, func is the solution to any issue, because the secret goal of this library is to provide a unique function func, that will perform a unified image processing.

Reply:
> Because func is short,

That's the main reason! it eases the formatting in the next line.
Since it is a local variable and that this code bloc will be removed in v0.19, I supposed that it is fine...